internal/olm/operator/registry/configmap/configmap_test.go

```diff
@@ -0,0 +1,236 @@
+package configmap
+
+import (
+	"crypto/sha256"
+	"encoding/base32"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/blang/semver"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/api/pkg/lib/version"
+	apimanifests "github.com/operator-framework/api/pkg/manifests"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestConfigmap(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Configmap Suite")
+}
+
+var _ = Describe("Testing configmap.go", func() {
+
+	Describe("Hash function", func() {
```

Comment:
name: `"hashContents"`. Basically I use the name of the function in the Describes.